src/cloud-api-adaptor/test/provisioner/trustee_kbs.go

```diff
@@ -497,20 +505,25 @@ func (p *KeyBrokerService) EnableKbsCustomizedAttestationPolicy(customizedOpaFil
 	return nil
 }
 
-func (p *KeyBrokerService) SetSampleSecretKey() error {
+func (p *KeyBrokerService) SetSecret(resourcePath string, secret []byte) error {
 	kbsClientDir := filepath.Join(trusteeRepoPath, "target/release")
 	privateKey := filepath.Join(getKbsKubernetesFilePath(), "base/kbs.key")
-	overlaysPath, err := getOverlaysPath()
+
+	tempDir, _ := os.MkdirTemp("", "kbs_resource_files")
+	defer os.RemoveAll(tempDir)
+
+	var secretFilePath = filepath.Join(tempDir, path.Base(resourcePath))
+	err := createAndWriteFile(secretFilePath, secret)
```

Comment:
nit: in this case it might be more concice to just use `os.WriteFile(secretFilePath, secret, 0644)`

Reply:
Thanks, I'll get rid of `createAndWriteFile` then